tool/lldb_tool.py

```diff
@@ -12,8 +12,109 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """A tool for LLM agents to interact within a LLDB."""
+import logging
+import os
+import subprocess as sp
+
+from experiment import oss_fuzz_checkout
+from experiment.benchmark import Benchmark
+from results import RunResult
 from tool.base_tool import BaseTool
 
+logger = logging.getLogger(__name__)
+
+# The directory in the oss-fuzz image
+JCC_DIR = '/usr/local/bin'
+
 
 class LLDBTool(BaseTool):
-  pass
+  """A tool for LLM agents to interact within a LLDB."""
+
+  def __init__(self, benchmark: Benchmark, name: str, project: str,
+               result: RunResult) -> None:
+    super().__init__(benchmark, name)
+    self.project = project
+    self.result = result
+    self.image_name = self._prepare_project_image()
+    self.container_id = self._start_docker_container()
+
+  def _prepare_project_image(self) -> str:
+    """Prepares the project's OSS-Fuzz docker image and returns the image name.
+    """
+    image_name = f'gcr.io/oss-fuzz/{self.project}'
+    # TODO(maoyi): implement image cache
+    if oss_fuzz_checkout.image_exists_locally(image_name, self.project):
+      logger.info('Using existing project image for %s', self.project)
+      return image_name
+    logger.info('Unable to find existing project image for %s', self.project)
+    command = [
+        'docker', 'build', '-t', image_name,
+        os.path.join(oss_fuzz_checkout.OSS_FUZZ_DIR, 'projects', self.project)
+    ]
+    try:
+      sp.run(command, cwd=oss_fuzz_checkout.OSS_FUZZ_DIR, check=True)
+      logger.info('Successfully build project image for %s', self.project)
+      return image_name
+    except sp.CalledProcessError:
+      logger.info('Failed to build image for %s', self.project)
```

Comment:
Again, would it be clearer to put this in `oss_fuzz_checkout`, or reuse some of its code?
Ideally, OSS-Fuzz-Gen, particularly agents, don't have to know OSS-Fuzz details.